neovim/.config/nvim/ftplugin/graphql.lua

```diff
@@ -1 +0,0 @@
-vim.bo.commentstring = '#%s'
```

Comment:
This got merged in `nvim-treesitter`. And FWIW I use `Comment.nvim` which has native support for `graphql`.